prog/dftb+/lib_dftb/dispdftd3.F90

```diff
@@ -261,4 +278,50 @@ function getRCutoff(this) result(cutoff)
 
   end function getRCutoff
 
+  ! D3H5 - additional H-H repulsion
+  !> Add the H-H repulsion for D3H5
+  subroutine addHHRepulsion(this, coords)
+    !> Instance of D3
+    class(DispDftD3), intent(inout) :: this
+    !> Current coordinates
+    real(dp), intent(in) :: coords(:,:)
+
+    integer :: i, j, c
+    real(dp) :: r, repE, dEdR, dCdR
+
+    ! Parameters (with conversion to a.u.)
+    real(dp) :: k = 0.30_dp * 0.001593601371772_dp
```

Comment:
Should use the constants.F90 parameters where possible, and add any missing conversion factors there.

Reply:
I've switched to conversion factors from constants.F90 everywhere.